tests/test_app.py

```diff
@@ -26,6 +26,19 @@ def test_data_request():
                         "chat": {"id": "chat-1"},
                         "project": {"id": "PROJECT-KEY"},
                         "user": {"id": "", "title": ""},
+                        "token_usage": {
```

Comment:
If we are declaring support of both the new and the old format, let's have a test for the old format as well.

Reply:
fixed